packages/shared/providers.yaml

```diff
@@ -466,6 +466,30 @@ contentstack:
     authorization_url: https://${connectionConfig.subdomain}.contentstack.com/apps/${connectionConfig.appId}/authorize
     token_url: https://${connectionConfig.subdomain}.contentstack.com/apps-api/apps/token
     docs: https://docs.nango.dev/integrations/all/contentstack
+#Untested configuration. Please reach out if you have a test account that we can use to test it.
```

Comment:
I'm not sure it nice to have this comment :D
Most likely the customer that will need it will reach out to us when time comes

Reply:
@bodinsamuel, thank you for pointing this out. @bastienbeurier had suggested we add this comment here.

Reply:
oh okay, miss communication then let's keep it that way ☺️

Reply:
I should add another pr to revert [this](https://github.com/NangoHQ/nango/pull/2234#discussion_r1625583374)☺️